python_modules/dagster/dagster/utils/log.py

```diff
@@ -239,3 +220,46 @@ def quieten(quiet=True, level=logging.WARNING):
     finally:
         if quiet:
             logging.disable(logging.NOTSET)
+
+
+def configure_loggers(handler: AllowedHandlers = "default", log_level="INFO"):
+    LOGGING_CONFIG = {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "colored": {
+                "()": coloredlogs.ColoredFormatter,
+                "fmt": default_format_string(),
+                "datefmt": default_date_format_string(),
+                "field_styles": {"levelname": {"color": "blue"}, "asctime": {"color": "green"}},
+                "level_styles": {"debug": {}, "error": {"color": "red"}},
+            },
+        },
+        "handlers": {
+            "default": {
+                "formatter": "colored",
+                "class": "logging.StreamHandler",
+                "stream": sys.stdout,
+                "level": log_level,
+            },
+            "null": {
+                "class": "logging.NullHandler",
+            },
+        },
+        "loggers": {
+            "dagster": {
+                "handlers": [handler],
+                "level": "INFO",
+            },
+            "dagit": {
+                "handlers": [handler],
+                "level": "INFO",
+            },
+        },
+    }
+
+    logging.config.dictConfig(LOGGING_CONFIG)
+
+    if handler == "default":
+        for logger in ["dagster", "dagit"]:
+            logging.getLogger(logger).handlers[0].formatter.formatTime = _mockable_formatTime
```

Comment:
does this need to be done here rather than in the config dictionary?

Reply:
There's a [`converter` attribute](https://docs.python.org/3/library/logging.html#logging.Formatter.formatTime), but unfortunately this can't be done in the config file.